tests/unit/clients/slack/test_slack_message_builder.py

```diff
@@ -323,19 +324,12 @@ def test_create_button_action_block(text, url):
 
 
 def test_prettify_and_dedup_list():
-    message_builder = SlackMessageBuilder()
-    list_prettified = message_builder.prettify_and_dedup_list(
-        ["name1", "name2", "name2"]
-    )
+    list_prettified = prettify_and_dedup_list(["name1", "name2", "name2"])
```

Comment:
Since you moved the method, can you also move the test to the correct location?

Reply:
Will do this tomorrow.

Reply:
Have way more time available then expected today, this is solved!